utils/bigclown/bigclown-gateway/files/config

```diff
@@ -0,0 +1,9 @@
+
+config gateway 'gateway'
+	option name 'usb-dongle'
+	option device '/dev/ttyUSB0'
```

Comment:
this is probably inappropriate

Reply:
Well that else should be there? It is a default configuration. I can comment it out if you want bit still it is a default value that in most cases is going to be correct.

You can see my respond to your next comment to see an idea of what it should be correctly according me.